ckanext/qa/templates/report/openness.html

```diff
@@ -97,7 +97,7 @@
             <td>{{ h.link_to(row['organization_title'], h.url_for('report.org', report_name='openness', organization=row['organization_name'])) }}</td>
           {% endif %}
           <td>{{ row['openness_score'] }}</td>
-          <td class="js-tooltip" title="{{ row['openness_score_reason'] }}">{{ h.truncate(row['openness_score_reason'], 150) }}</td>
+          <td class="js-tooltip" title="{{ row['openness_score_reason'] }}">{{ row['openness_score_reason']|truncate(150) if row['openness_score_reason'] }}</td>
```

Comment:
Does this null check work correctly if the key doesn't exist on the object at all?

Reply:
I don't know if it would, but if it doesn't then I don't think the row title would work either.